doc/development/guide/pullrequests.rst

```diff
@@ -1,74 +1,58 @@
 Submitting a pull request
 =========================
 
-Before submitting a pull request, please make sure the following is done:
 
-* **All new features must include a unit test.** If you've fixed a bug or added
```

Comment:
How come these were removed?

Reply:
I mention all new features need to be fully tested, just later.  I don't explicitly specify "unit test", since unit tests are necessary but not sufficient.  Depending on the feature, it may need integration and functional tests too.